cmd/loadtest/server.go

```diff
@@ -0,0 +1,162 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/mattermost/mattermost-load-test-ng/config"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/control"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/control/simplecontroller"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/store/memstore"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/user/userentity"
+	"github.com/mattermost/mattermost-server/mlog"
+	"github.com/spf13/cobra"
+)
+
+type Agent struct {
+	loadTests map[string]*loadtest.LoadTester
+}
```

Comment:
To me the `Agent` is really the `LoadTester` which might actually need renaming.
I would call this type simply `API` and keep the router here (similar to what happens in the server repository). 
Also I think all this code (except the cmd part) should live in a dedicated `api` or `server` package in the project's root (alongside loadtest and cmd dirs).